vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -332,7 +332,7 @@ - (void)getCreateScript:(ledger::GetCreateScriptCallback)callback
     case BATLedgerDatabaseMigrationTypeNone:
       // We shouldn't be migrating, therefore doesn't make sense that
       // `getCreateScript` was called
-      BLOG(ledger::LogLevel::LOG_ERROR) << "DB: Attempted CoreData migration with an empty migration script" << std::endl;
+      BLOG(0) << "DB: Attempted CoreData migration with an empty migration script" << std::endl;
```

Comment:
I notice you've been using `0` a lot but our previous log severity levels started at 1 (`LOG_ERROR`). Does this mean we're just shifting everything down? Where 0 = least verbose (errors) and as we get hire we get more and more verbose?

Reply:
0 is most verbose, and the higher the number the less verbose

Reply:
yup 0 is error now